Add wueadress extension

janschlatter opened the pull request:
## Description

<!-- A summary of your change. If you add a new extension or command, explain what it does. -->

wueAdress provides an interface for looking up contact infos for the University of Würzburg. It uses the official address page to acquire this info, it's all public info. It uses actions to streamline connecting to colleagues.

## Screencast

https://user-images.githubusercontent.com/63644885/231805547-2b509c90-64f2-4a8a-b88f-6aeb69cddf21.mp4

Please excuse the messy background.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

pernielsentikaer commented on extensions/wueadress/src/index.tsx:
```diff
@@ -0,0 +1,160 @@
+      isLoading={isLoading}
+      onSearchTextChange={setSearchQuery}
+      searchBarPlaceholder="Search for a contact..."
+      navigationTitle="WueAddress"
```
```suggestion
```
Don't change the navigationTitle in the root command - it will be automatically set to the command name. Use navigationTitle only in nested screens to provide additional context.

janschlatter replied:
Made changes in commit

pernielsentikaer reviewed (changes requested):
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

* The extension logo doesn't fit well with the [dark theme](https://developers.raycast.com/api-reference/user-interface/icons-and-images#image.imagelike), could you add a light version of the icon too?
  
<img width="1000" alt="Raycast 2023-04-20 at 14 45 46" src="https://user-images.githubusercontent.com/38033107/233370357-faea3598-6a22-4da5-9e6a-8eb2d5795f8f.png">

* Same issue with the no-view image, it's not possible to see in the [dark theme](https://developers.raycast.com/api-reference/user-interface/icons-and-images#image.imagelike) and should also be resized to around 128x128

![Raycast-88M7pGW6@2x](https://user-images.githubusercontent.com/38033107/233370688-dd40a7a2-bdeb-4b8a-bb09-b6766e9957c4.png)


I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

janschlatter commented on extensions/wueadress/src/index.tsx:
```diff
@@ -0,0 +1,160 @@
+
+  return (
+    <List
+      isLoading={isLoading}
```
I compared performance with and without trottle and it's considerably slower – Considering this extension is quite niche, I'd like to keep throttling off for time being.

janschlatter commented on extensions/wueadress/src/index.tsx:
```diff
@@ -0,0 +1,160 @@
+            actions={
+              <ActionPanel>
+                {result.emails.map((email) => (
+                  <Action.Open key={email} icon="AtSymbol" title={`Write Email`} target={`mailto:${email}`} />
```
Made changes in commit

janschlatter commented on extensions/wueadress/src/index.tsx:
```diff
@@ -0,0 +1,160 @@
+import { ActionPanel, Action, List } from "@raycast/api";
```
Made changes in commit

janschlatter commented:
Hey @pernielsentikaer, 

thank you so much for your thorough code review! I addressed all of your points and addressed it in my last commit. I also changed the icon and image so it works with dark mode – something I forgot to check. Thanks again for your work.

pernielsentikaer commented:
Seems like the [List.EmptyView](https://developers.raycast.com/api-reference/user-interface/list#list.emptyview) icon is still the old one, right?

![Raycast-auTcjzsi@2x](https://user-images.githubusercontent.com/38033107/234855066-f6fa7aad-6b67-4808-b726-71018b5eef6b.png)
